### ISSUE TITLE
Is possible to converting a .pt PyTorch model to a .pth file ?

### ISSUE THREAD

**Issue by chathuryaprasad** [ultralytics/yolov5#13478]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Is possible to converting a .pt PyTorch model to a .pth file ?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @CPrasa, thank you for your interest in YOLOv5 🚀! Converting a `.pt` PyTorch model to a `.pth` file is an interesting question. 

If this is a ❓ Question, please provide as much detail as possible about your use case or requirements. This will help us provide better guidance. If this is a 🐛 Bug Report, we kindly request a **minimum reproducible example (MRE)** to help us debug and assist you more effectively.

## Requirements

Ensure you are using **Python>=3.8.0** and have all required dependencies installed, including **PyTorch>=1.8**. To set up YOLOv5:
```bash
git clone the repository from the source  # clone
cd into the yolov5 directory
pip install required dependencies listed in requirements.txt
```

## Environments

YOLOv5 can run on various verified environments including Jupyter Notebooks with free GPU runtime, cloud platforms like Google Cloud or AWS, or in a Docker environment with pre-installed dependencies.

## Status

You can check the status of YOLOv5's integrations, including training, validation, and inference workflows, to ensure functionality is operational.

This is an automated response to help guide you, but no worries—a human Ultralytics engineer will take a closer look and assist you further soon! 😊

**Comment by pderrenger** [comment]
@CPrasa yes, a `.pt` PyTorch model can be converted to a `.pth` file as both formats are interchangeable and represent serialized PyTorch models. You can achieve this by loading the `.pt` model using PyTorch and saving it directly as a `.pth` file using `torch.save`. Here's an example:

```python
import torch

# Load the .pt model
model = torch.load('model.pt')

# Save as .pth
torch.save(model, 'model.pth')
```

Let me know if you encounter any issues!
